test/mocha-global.js

```diff
@@ -1,10 +1,19 @@
-import { after } from 'mocha'
+import { before, after } from 'mocha'
 import { cleanupTestSpaces } from './helpers'
 
-after('clean up test spaces', async () => {
+const housekeeping = async () => {
   try {
     await cleanupTestSpaces()
   } catch {
     // ignore if the cleanup fails. Usually fails locally due to missing credentials
```

Comment:
Should we log the error? This might help users to learn how to fix it

Reply:
While I agree with your points Meike I'd still be in favor of logging the error. An error that is not logged (especially if it also CAN be something else than a credential error) creates the risk of significant confusion/effort when needing to debug a problem in the future.

Reply:
Okay, added some logging :-)